full-service/src/json_rpc/json_rpc_request.rs

```diff
@@ -66,11 +66,12 @@ pub enum JsonCommandRequest {
     },
     build_and_submit_transaction {
         account_id: String,
-        addresses_and_values: Option<Vec<(String, String)>>,
+        addresses_and_amounts: Option<Vec<(String, Amount)>>,
         recipient_public_address: Option<String>,
-        value_pmob: Option<String>,
+        amount: Option<Amount>,
         input_txo_ids: Option<Vec<String>>,
-        fee: Option<String>,
+        fee_value: Option<String>,
```

Comment:
For this one, I would suggest keeping to one param, and having a backward compatible param

```suggestion
        // Multi-token Support
        fee_amount: Option<Amount>,
        // MOB-Only
        fee: Option<String>,
```